packages/lib/src/vis/line/LineVis.tsx

```diff
@@ -152,14 +153,33 @@ function LineVis(props: Props) {
                 {`${abscissaLabel ?? 'x'} = ${formatTooltipVal(abscissa)}`}
 
                 <div className={styles.tooltipValue}>
-                  <strong>{formatTooltipVal(value)}</strong>
-                  {error && ` ±${formatTooltipErr(error)}`}
-                  {dtype && <em>{` (${formatNumType(dtype)})`}</em>}
+                  {auxiliaries.length > 0 && (
+                    <span
+                      className={styles.mark}
+                      style={{ color: curveColor }}
+                    />
+                  )}
+                  <span>
+                    <strong>{formatTooltipVal(value)}</strong>
+                    {error && ` ±${formatTooltipErr(error)}`}
+                    {dtype && <em>{` (${formatNumType(dtype)})`}</em>}
+                  </span>
                 </div>
 
-                {auxiliaries.map(({ label, array }) => {
+                {auxiliaries.map(({ label, array }, index) => {
                   const val = formatTooltipVal(array.get(xi));
-                  return <div key={label}>{`${label} = ${val}`}</div>;
+                  const color = auxColors[index] || lastAuxColor;
+
+                  return (
+                    <div
+                      className={styles.tooltipAux}
+                      key={label}
+                      data-color={color}
+                    >
+                      <span className={styles.mark} style={{ color }} />
+                      {label} = {val}
```

Comment:
It makes sense to me to format auxiliary values as the signal value
```suggestion
                      {label} = {formatTooltipVal(val)}
```

Reply:
This is already the case, but I've removed the extra `val` variable for simplicity.

Reply:
Oh, indeed, sorry for the false alarm